telemetry/telemetry.go

```diff
@@ -33,6 +33,7 @@ var (
 	gnmi_translib_write = flag.Bool("gnmi_translib_write", gnmi.ENABLE_TRANSLIB_WRITE, "Enable gNMI translib write for management framework")
 	gnmi_native_write   = flag.Bool("gnmi_native_write", gnmi.ENABLE_NATIVE_WRITE, "Enable gNMI native write")
 	threshold         = flag.Int("threshold", 100, "max number of client connections")
+	withMasterArbitration = flag.Bool("with-master-arbitration", false, "Enables master arbitration policy.")
```

Comment:
How an admin can enable `--with-master-arbitration` flag? I did not find any PR to update the telemetry service startup script

Reply:
*My original comment got truncated for some reason.. Following is the continuation of it:*

What will happen if client sends MA extension and server is not started with `--with-master-arbitration` flag? Current implementation ignores the MA extension and accepts all set requests. Clients cannot discover that  that MA is off and we may enter an incorrect state should a management network partition happen.

I feel we do not need this flag. The spec already gives full control to the clients to use MA or not. Set requests will be accepted if client did not send a MA extension. So, legacy clients will continue to work as before. This flag is redundant.